src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -372,6 +382,14 @@
     type: string
     short-summary: Load balancer outbound IP prefix resource IDs.
     long-summary: Comma-separated public IP prefix resource IDs for load balancer outbound connection. Valid for Standard SKU load balancer cluster only.
+  - name: --load-balancer-outbound-ports
+    type: string
+    short-summary: Load balancer outbound allocated ports.
```

Comment:
Will update to `Desired static number of outbound ports per VM in the load balancer backend pool. By default, set to 0 which will use the default allocation table.`

Reply:
Desired static number of outbound ports per VM in the load balancer backend pool. By default, set to 0 which uses the default allocation based on the number of VMs